diagnostics/ENSO_MSE/shared/get_dimensions.py

```diff
@@ -0,0 +1,32 @@
+import os.path
+import sys

```

Comment:
YOu could write a more general script that combines get_dimensions, get_lat_lon_plevels_in, get_parameters_in. and get_season to read any variable from the desired .txt file by adding arguments for the filepath, a list of variables to read, and maybe using [readlines()](https://www.pythontutorial.net/python-basics/python-read-text-file/) to return a list of strings. You could then loop through the list for spaces, commas, or however you are separating the columns and assign them to a specified variable.